dremio_client/odbc.py

```diff
@@ -28,7 +28,7 @@
 
 
 _WINDOWS_DRIVER = "Dremio Connector"
-_OSX_DRIVER = "Dremio ODBC Driver"
+_OSX_DRIVER = "/Library/Dremio/ODBC/lib/libdrillodbc_sbu.dylib" # had to add this for the driver path
```

Comment:
what happens if this is not changed? I thought the purpose of this was to fix #66 so that odbc worked w/o having to give absolute paths?

Reply:
@rymurr the driver can not be found if this is not changed

Reply:
Yes.  You are right.  I was doing that on my end because `pyodbc.drivers()` returned an empty list so I included the full path for the Driver to get it to work.